zc_plugins/EmailArchiveManager/v4.0.0/admin/email_archive_manager.php

```diff
@@ -33,11 +39,17 @@ function zen_get_email_archive_search_query(string $search_text, string $sd_raw,
         'email_text',
         'date_sent',
         'module',
-        'errorinfo',
+        'errorinfo'
     ];
     $where_clauses = [];
 
-    $archive_search_sql = 'SELECT ' . implode(', ', $select_fields) . ' FROM ' . TABLE_EMAIL_ARCHIVE . " \n";
+    if (!empty($error_keywords)) {
```

Comment:
I think this `if(!empty($error_keywords))` is redundant now, right?

Reply:
No, if $error_keywords is empty then we do not want to include an unnecessary LIKE operator on that TEXT column.

Reply:
Well, `$error_keywords` isn't a variable present in the scope of this function.

I think the search "works" because I'd included `errorinfo` in the array on line 66 below.

Reply:
Oh, I'm being an idiot. Right.

Reply:
Fix pushed.